src/adapters/analytics/AnalyticsAdapter.js

```diff
@@ -71,40 +74,63 @@ export default function AnalyticsAdapter({ url, analyticsType, global, handler }
   function _enable(config) {
     var _this = this;
 
-    //first send all events fired before enableAnalytics called
-    events.getEvents().forEach(event => {
-      if (!event) {
-        return;
-      }
+    if (typeof config === 'object' && typeof config.options === 'object') {
+      _sampled = typeof config.options.sampling === 'undefined' || Math.random() < parseFloat(config.options.sampling);
+    } else {
+      _sampled = true;
+    }
 
-      const { eventType, args } = event;
 
-      if (eventType === BID_TIMEOUT) {
-        _timedOutBidders = args.bidderCode;
-      } else {
-        _enqueue.call(_this, { eventType, args });
-      }
-    });
+    if (_sampled) {
```

Comment:
Assuming this is only `falsy` when `config.options.sampling` === 0. 
Why would you want to wire up analytics not to send data? I guess I'm saying there is no point for this.